kairos-server/src/routes/transfer.rs

```diff
@@ -35,12 +36,14 @@ pub async fn transfer_handler(
 
     tracing::info!("queuing transaction for trie update");
 
+    let transfer = KairosTransaction::Transfer(Signed {
+        public_key,
+        nonce,
+        transaction: transfer,
+    });
+    db::insert(state.pool.clone(), transfer.clone()).await?;
```

Comment:
Let's put the db access behind a feature flag so that we don't need postgres to run simple tests.

Reply:
Done.